languages/csharp/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+You have four tasks, all related to the time spent cooking the lasasgna.
+
+### 1. Define the expected oven time in minutes
+
+Define the `Lasagna.ExpectedMinutesInOven` method that does not take any parameters and returns how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
```

Comment:
Also, I'm not sure the convention for C# (so this may be waffle 😄), but for other languages when referring to methods, the parenthesis are included. So `Lasagna#ExpectedMinutesInOven()` etc.

Reply:
Good suggestion. I'd like to apply it to all exercises at once, so I've created [this issue](https://github.com/exercism/v3/issues/1041).